distributionviewer/api/tests/test_views.py

```diff
@@ -14,24 +14,26 @@
 
 class TestMetric(TestCase):
 
-    def create_data(self, date=None):
+    def create_data(self, date=None, population=None):
+        date = date or datetime.date(2016, 1, 1)
+        population = population or 'All'
```

Comment:
Just out of curiosity, why this pattern instead of making `datetime.date(2016, 1, 1)` and `'All'` default arguments?